presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -131,6 +131,7 @@
     private boolean bucketExecutionEnabled = true;
     private boolean sortedWritingEnabled = true;
     private boolean ignoreTableBucketing;
+    private int minBucketCountToNotIgnoreTableBucketing;
```

Comment:
nit: How about `minNumberOfBucketsForBucketedExecution`

CC: @aweisberg ?

Reply:
I considered this one before as it is simpler than `minBucketCountToNotIgnoreTableBucketing`. However, when bucket count is less than the threshold, there is still bucketed execution with a larger bucket count.

Reply:
> minBucketCountToNotIgnoreTableBucketing

This a little bit hard to read because of kinda "double negative".

> minNumberOfBucketsForBucketedExecution

Should be easier to read. It defines the minimum number of buckets for the bucketed execution to be enabled. 

e.g.:

if `minNumberOfBucketsForBucketedExecution=1024` and the table has only 512 buckets, it doesn't meet the requirement and the bucketed execution will be disabled. If the table has 2048 buckets it meets the requirement, but the bucketed execution can still be manually disabled with the session property.

Reply:
Since the actual execution depends on other session property and whether LBM is enabled, how about `minNumberOfBucketsForBucketedTable`,  if bucket count is less than the threshold, this table is considered as not bucketed table.

Reply:
@aweisberg What do you think?

Reply:
I think `minNumberOfBucketsForBucketedExecution` is clearer `minNumberOfBucketsForBucketedTable` because the table is still bucketed we are just not using an execution plan that leverages it.

Reply:
Based on the offline discussion we've decided to proceed with the existing name